pyiron_atomistics/atomistics/structure/analyse.py

```diff
@@ -63,6 +67,7 @@ def get_layers(self, distance_threshold=0.01, id_list=None, wrap_atoms=True, pla
                 description: sklearn.cluster.AgglomerativeClustering
             id_list (list/numpy.ndarray): List of atoms for which the layers
                 should be considered.
+            wrap_atoms: TODO.
```

Comment:
```suggestion
            wrap_atoms (bool): Whether to consider periodic boundary conditions according to the box definition or not. If set to `False`, atoms lying on box boundaries are considered to belong to different layers, regardless of whether the box itself has the periodic boundary condition in this direction or not. If `planes` is not `None` and `wrap_atoms` is `True`, this tag has the same effect as calling `get_layers()` after calling `center_coordinates_in_unit_cell()`
```

Reply:
This might be too long for this section - maybe the second part (starting from *If set to*) should be added as a comment at the end of the Doc String.

Reply:
Perfect, thanks. I wouldn't worry about it too much; because of the indentation formatting people should be able to skip over this parameter while reading pretty easily if they're not interested.

Reply:
Sorry one last thing: Instead of *atoms lying on box boundaries*, maybe better *atoms lying on **opposite** box boundaries*.

Reply:
👍 Done